cmake/zig-build.cmake

```diff
@@ -37,26 +38,33 @@ else()
     message(FATAL_ERROR "Unsupported platform")
 endif()
 
-if(NOT EXISTS "${CMAKE_BINARY_DIR}/zig-relsafe-espressif-${TARGET_ARCH}-${TARGET_PLATFORM}-baseline")
-    file(DOWNLOAD "https://github.com/kassane/zig-espressif-bootstrap/releases/download/0.14.0-xtensa-dev/zig-relsafe-espressif-${TARGET_ARCH}-${TARGET_PLATFORM}-baseline.${EXT}"
-        "${CMAKE_BINARY_DIR}/zig.${EXT}")
+if (CONFIG_IDF_TARGET_ARCH_XTENSA OR CONFIG_IDF_TARGET_ESP32P4)
```

Comment:
on CI (c3/c6) get error: `/bin/sh: 1: zig: not found`

Reply:
Suggestion: add URL of zig-upstream tarballs.
For those who already have Zig installed on their system, you could use CMake Option (like `-DZIG_INSTALLED_SYSTEM`).

Reply:
After the release of v0.14.0 (upstream), zig-xtensa will also release the final Release and both used the same build (changing only the LLVM backend)